examples/plot_skorch_trial_decoding.py

```diff
@@ -24,10 +24,14 @@
 from torch import optim
 from torch.utils.data import Dataset
 
+from sklearn.metrics import f1_score
+
 from skorch.net import NeuralNet
 
 from braindecode.models import ShallowFBCSPNet
 from braindecode.util import set_random_seeds
+from braindecode.experiments.classifier import BraindecodeClassifier
```

Comment:
As I would expect everyone to use an EEGClassifier object maybe we can put this in the root namespace.

from braindecode import EEGClassifier

wdyt?